astroplan/plots/sky.py

```diff
@@ -0,0 +1,179 @@
+    style_kwargs.setdefault('color', 'k')
+
+    # Grab altitude and azimuth from Astroplan objects.
+    # Note that values must be made dimensionless before plotting.
+    # Modifying altitude is easier than inverting r-axis.
+    altitude = (91 * u.deg - observer.altaz(time, target).alt) *(1/u.deg)
+    # Azimuth MUST be given to plot() in radians.
+    azimuth = observer.altaz(time, target).az * (1/u.deg) * (np.pi/180.0)
+
+    # Some checks & info for labels.
+    if not hasattr(target, 'name'):
+        target_name = '?'
+    else:
+        target_name = target.name
+    style_kwargs.setdefault('label', target_name)
+
+    # We only want to plot positions above the horizon.
+    az_plot = None
+    for alt in range(0, len(altitude)):
+        if altitude[alt] > 91.0:
+            print("Warning: Target " + str(target_name) +
+                  " is below the horizon at time: " + str(time[alt]))
+        else:
+            if az_plot is None:
+                az_plot = np.array([azimuth[alt]])
+            else:
+                az_plot = np.append(az_plot, azimuth[alt])
+    alt_plot = altitude[altitude <= 91.0]
+    if az_plot is None:
+        az_plot = []
+
+    # More axes set-up.
+    # Position of North (azimuth = 0).
+    if north == 'top':
+        n_dir = 'N'
+    elif north == 'left':
+        n_dir = 'W'
+    elif north == 'bottom':
+        n_dir = 'S'
+    elif north == 'right':
+        n_dir = 'E'
+    else:
+        print("Warning: You have entered an invalid position for North."
+              + " Defaulting to 'top'.")
+        n_dir = 'N'
+    ax.set_theta_zero_location(n_dir)
+
+    # Direction of azimuth increase.
+    if inc_az_ccw is False:
+        ax.set_theta_direction(-1)
+
+    # Plot target coordinates.
+    # ax.plot connects dots.
+    #ax.plot(az_plot, alt_plot, **style_kwargs)
+    # ax.scatter does not connect dots.
+    ax.scatter(az_plot, alt_plot, **style_kwargs)
+    #ax.polar(az_plot, alt_plot, **style_kwargs)
+
+    # Set radial limits.
+    ax.set_rlim(1, 91)
+
+    # Grid, ticks & labels.
+    # May need to set ticks and labels AFTER plotting points.
+    ax.grid(True, which='major')
+    degree_sign = u'\N{DEGREE SIGN}'
+
+    # For positively-increasing range (e.g., range(1, 90, 15)),
+    # labels go from middle to outside.
+    r_labels = [
+        '90' + degree_sign,
+        '',
+        '60' + degree_sign,
+        '',
+        '30' + degree_sign,
+        '',
+        '0' + degree_sign + ' Alt.',
+        ]
+    theta_labels = [
+        'N' + '\n' + '0' + degree_sign + ' Az.',
+        '45' + degree_sign,
+        'E' + '\n' + '90' + degree_sign,
+        '135' + degree_sign,
+        'S' + '\n' + '180' + degree_sign,
+        '225' + degree_sign,
+        'W' + '\n' + '270' + degree_sign,
+        '',
+        ]
+
+    # Set ticks and labels.
+    ax.set_rgrids(range(1, 106, 15), r_labels, angle=-45)
+    ax.set_thetagrids(range(0, 360, 45), theta_labels)
+
```

Comment:
I think you will need to rotate the `theta_labels` for alternative plots (S=0, E/W flipped) if you are going to use this approach, because it is distributing them from 0->360 in +45 increments.  Alternatively, you could specify a different range and/or flip the sign of the intermediate value.